scripts/update-apk.sh

```diff
@@ -45,6 +42,14 @@ for file in app*; do
     fi
 done
 
+if [[ $TRAVIS_BRANCH =~ ^(master)$ && $TRAVIS_PULL_REQUEST_BRANCH =~ ^(development)$ && $TRAVIS_REPO_SLUG =~ ^('fossasia/open-event-attendee-android')$ && $TRAVIS_PULL_REQUEST_SLUG =~ ^('fossasia/open-event-attendee-android')$ ]];then
+    echo "cool, so you're trying to release a new version. Lets check things out first, hold on..."
+    cd ..
+    gem install fastlane
+    fastlane supply --aab ./apk/eventyay-attendee-master-app-playStore-release.aab --skip_upload_apk true --track alpha --json_key ./scripts/fastlane.json --package_name $PACKAGE_NAME --validate_only true
+    exit 0
+fi
```

Comment:
Unnecessary duplicate code. 

Conditionally define a command

```sh
if DRY_RUN {
	export FASTLANE_DRY_RUN=--validate_only true
}
```

Then just append `FASTLANE_DRY_RUN` to the end of the fastlane command below `fastlane ... $PACKAGE_NAME $FASTLANE_DRY_RUN`

Reply:
got it

Reply:
@iamareebjamal ok so, If I dont duplicate this we will end up pushing to apk branch. currently we are terminating the script as soon as we validate, If I need to follow this I should seperate the sections inside the script into an ifelse condition. Let me know if you want me to go ahead with that approach

Reply:
Move the fastlane block up and exit before branch pushing if it is dry run mode